homeassistant/components/tibber/sensor.py

```diff
@@ -377,20 +372,21 @@ async def _fetch_data(self):
         ]["estimatedAnnualConsumption"]
 
 
-class TibberSensorRT(TibberSensor):
+class TibberSensorRT(TibberSensor, update_coordinator.CoordinatorEntity):
     """Representation of a Tibber sensor for real time consumption."""
 
-    _attr_should_poll = False
     entity_description: TibberSensorEntityDescription
 
     def __init__(
         self,
         tibber_home,
         description: TibberSensorEntityDescription,
         initial_state,
+        coordinator: TibberRtDataCoordinator,
     ):
         """Initialize the sensor."""
-        super().__init__(tibber_home)
+        update_coordinator.CoordinatorEntity.__init__(self, coordinator)
+        TibberSensor.__init__(self, tibber_home)
```

Comment:
A future improvement would be to make the `TibberSensor` class support multiple inheritance properly by calling `super().__init__` in its `__init__` method with the parameters that it received and doesn't support. Something like this:

```py
def __init__(self, *args, tibber_home, **kwargs):
    super().__init__(*args, **kwargs)
    self._tibber_home = tibber_home
    ...
```
Then we can change here to:
```py
super().__init__(coordinator=coordinator, tibber_home=tibber_home)
```